src/sources/util/http.rs

```diff
@@ -124,6 +126,53 @@ impl HttpSourceAuth {
     }
 }
 
+fn decode(header: &Option<String>, mut body: Bytes) -> Result<Bytes, ErrorMessage> {
+    if let Some(encodings) = header {
+        for encoding in encodings.rsplit(',').map(str::trim) {
```

Comment:
I don't see a test for having multiple values here, did I miss it?

Reply:
It's called `http_gzip_deflate` in `http` source file.